python/tskit/trees.py

```diff
@@ -10171,3 +10453,30 @@ def write_ms(
                         )
             else:
                 print(file=output)
+
+@dataclass
+class PCAResult:
+    """
+    The result of a call to TreeSequence.pca() capturing the output values
+    and algorithm convergence details.
+
+
+    """
+    factors: np.ndarray
```

Comment:
We need to define the dimensions here. Currently the first dimension is samples/individuals and the second is the num_componnents. Is there a strong reason for doing it this way? I expected it to be ``(num_components, num_samples)`` as you usually want to access all the values for a given component together?

Reply:
Aha, it's following scikit learn. That's an excellent guide to follow - let's just do what scikit learn does and make our API as compatible with them as possible? We should document this as a stated goal.